manage-indexes/create-indexes-basic.ipynb

```diff
@@ -61,7 +66,20 @@
     ")\n",
     "\n",
     "result = collection.create_search_index(model=search_index_model)\n",
-    "print(result)"
+    "print(\"New search index named \" + result + \" is building.\")\n",
+    "\n",
+    "# Wait for initial sync to complete\n",
+    "print(\"Polling to check if the index is ready. This may take up to a minute.\")\n",
+    "predicate=None\n",
+    "if predicate is None:\n",
+    "  predicate = lambda index: index.get(\"queryable\") is True\n",
+    "\n",
+    "while True:\n",
+    "  indices = list(collection.list_search_indexes(name))\n",
```

Comment:
Looks like `name` is not defined here, so I guess we need to use `result` or define `name`.
```suggestion
    "  indices = list(collection.list_search_indexes(result))\n",
```